allauth/socialaccount/providers/github/views.py

```diff
@@ -30,9 +30,11 @@ def complete_login(self, request, app, token, **kwargs):
         )
         resp.raise_for_status()
         extra_data = resp.json()
-        if app_settings.QUERY_EMAIL and not extra_data.get("email"):
```

Comment:
There is one other provider, BitBucket, that does this pattern, we may want to also make this change there: https://github.com/mecampbellsoup/django-allauth/blob/aa89da5ce199e71b0ee3fe3a2e44007de1d55ea6/allauth/socialaccount/providers/bitbucket_oauth2/views.py#L24

Reply:
Would be good indeed, can you make that change as part of this PR?